jms/src/main/scala/akka/stream/alpakka/jms/Jms.scala

```diff
@@ -18,25 +18,37 @@ sealed trait Destination
 final case class Topic(name: String) extends Destination
 final case class Queue(name: String) extends Destination
 
-final case class JmsTextMessage(body: String, properties: Map[String, Any] = Map.empty) {
+final case class JmsTextMessage(body: String, _type: Option[String] = None, properties: Map[String, Any] = Map.empty) {
```

Comment:
Make it `type` or `jmsType` instead. Underscore is used for private fields and this is a public field.

Reply:
I agree, but I can't call it type (type is reserved) and jmsType is redundant with the name of the class, don't you think? Let's say I'll call it that way

Reply:
I will also add some test, right!